game.cpp

```diff
@@ -0,0 +1,137 @@
+#include <iostream>
+#include "game.h"
+#include "constants.h"
+
+Game::Game()
+{
+    this->m_current_player = 0;
+    this->m_grid = new int*[SIZE];
+    
+    for (int i = 0; i < SIZE; ++i)
+        this->m_grid[i] = new int[SIZE];
+}
+
+void Game::start()
```

Comment:
Usually you put all the methods in the main class - it makes it much more readable.